merfi/backends/gpg.py

```diff
@@ -58,3 +72,30 @@ def sign_release(self, path):
             logger.info('signing: %s' % path)
             util.run(detached)
             util.run(clearsign)
+
+    def sign_repomd(self, path):
+        """ Sign a Yum repomd.xml file. """
+        if merfi.config.get('check'):
+            new_md_path = path + '.asc'
+            logger.info('[CHECKMODE] signing: %s' % path)
+            logger.info('[CHECKMODE] signed: %s' % new_md_path)
+            return
+        os.chdir(os.path.dirname(path))
+        detached = ['gpg', '--batch', '--yes', '--armor', '--detach-sig',
+                    '--output', 'repomd.xml.asc', 'repomd.xml']
+        logger.info('signing: %s' % path)
+        util.run(detached)
+
+    def sign_rpm(self, rpm):
+        """ Sign a .rpm file. """
+        if merfi.config.get('check'):
+            logger.info('[CHECKMODE] signed: %s' % rpm.name)
+            return
+        rpmsign = ['rpm',
+                   '--define', '_signature gpg',
+                   '--define', '__gpg /usr/bin/gpg',
+                   '--define', '__gpg_check_password_cmd /bin/true',
+                   '--define', '__gpg_sign_cmd %{__gpg} gpg --batch --no-verbose --no-armor --no-secmem-warning -u "%{_gpg_name}" -sbo %{__signature_filename} %{__plaintext_filename}',  # NOQA: E501
+                   '--addsign', rpm.path]
+        logger.info('signing: %s' % rpm.path)
```

Comment:
Same here I think, it would be useful to have these available as part of CHECKMODE

Reply:
good suggestion, I've fixed this as well.